.travis.yml

```diff
@@ -18,13 +18,19 @@ git:
 
 go_import_path: sigs.k8s.io/kubebuilder
 
+services:
+- docker
```

Comment:
Check if we can enable caching for docker images ? will make it run faster.

Reply:
Travis document doesn't recommend caching docker images: https://docs.travis-ci.com/user/caching/#things-not-to-cache